.gitignore

```diff
@@ -25,3 +25,4 @@ bin/
 run/
 
 *.ase
+/radle-version/
```

Comment:
What is this line for?

Reply:
You can remove it I was excluding a folder I renamed while upgrading gradle.